packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -659,6 +663,56 @@ at the end of the document.
 Note that limiting documents in this way means that they are no longer correctly
 formatted JSON objects.
 
+
+[[configuration-cassandra]]
+==== Cassandra Configuration Options
+
+The following settings are specific to the Cassandra protocol. Here is a sample
+configuration for the `cassandra` section of the +{beatname_lc}.yml+ config file:
+
+[source,yaml]
+------------------------------------------------------------------------------
+packetbeat.protocols.cassandra:
+  ports: [9042]
+  send_request: true
+  send_request_header: true
+  send_response: true
+  send_response_header: true
+  compressor: "snappy"
+  ignored_ops: ["SUPPORTED","OPTIONS"]
+------------------------------------------------------------------------------
+
+
+===== send_request
```

Comment:
like `ports, `send_request` and `send_respose` is already part of common protocol options (already documented)